src/index.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState, useEffect } from 'react'
+import ReactDOM from 'react-dom'
+
+const Options = () => {
+    const [webhookUrl, setWebhookUrl] = useState('')
+    const handleChange = (e) => setWebhookUrl(e.target.value) // TODO: handle url validation
+    const handleClick = (e) => {
+        e.preventDefault()
+        console.log(`Storing: ${webhookUrl}`)
```

Comment:
Since the console is seemingly public facing, it's probably best that we don't print out the `webhookUrl` since it gives the bearer authorization to post messages to a Discord channel. But it also looks like the [console can't be read by other javascript programs](https://stackoverflow.com/questions/19846078/how-to-read-from-chromes-console-in-javascript) so we might be good.

Reply:
No problem if the user sees it, since they've already copied it. 

And that's right about the logs, those are considered side effects so we should be ok. Besides I don't think we're going to keep this log there for very long anyway.